customer-config/scan-and-go/sng.device-api-settings.v1.json

```diff
@@ -10,6 +10,13 @@
       "minimum": 0,
       "maximum": 100,
       "description": "The minimum battery level required to release a handheld scanner."
+    },
+    "hhsInactiveDaysBeforeRemoval": {
+      "type": "integer",
+      "default": 90,
+      "minimum": 0,
+      "maximum": 180,
+      "description": "The amount of days after which inactive handheld scanner is removed. Default value: 90"
```

Comment:
Would you change the description to 
"Number of days after which an inactive handheld scanner is removed. The default is set to 90 days."

and please merge after this change if someone approves